### DISCUSSION TITLE
Is there a way to make kitty terminal read-only (like GNOME terminal)?

### DISCUSSION THREAD

**Discussion by siddhpant** [kovidgoyal/kitty#7457] [Q&A] [answered]
I think basically no input gets passed through.

Here is the option shown upon right-click on GNOME terminal:

![image](https://github.com/kovidgoyal/kitty/assets/25429745/7620cac4-b4eb-4257-8fac-190c7cc36a71)

Also, is there any way to get a similar menu on right-click?

**Comment by kovidgoyal** [comment, marked as answer]
No kitty has no read only mode not sure why you would want one.
Presumably you are trying to view some log, so instead of blocking input
to the log viewing program simply use one that doesnt accept keyboard
input like say cat.

And no there is no right click menu in kitty.

**Reply by siddhpant** [reply]
Thanks!

Not really log, sometimes I want to just have stuff there so I don't change anything anyhow, like having to show something or few times just cleaning keyboard or moving around. Currently I'm just using the pager mode for it which is great, but still it accepts input.
